src/shared/ui/survey/MediaItems/types.ts

```diff
@@ -0,0 +1,9 @@
+type MediaValue = {
+  uri: string;
+  filename: string;
```

Comment:
Does this naming come somewhere from package or some other place that doesn't depend on us? If no, we better rename to file**N**ame.

Reply:
@vmkhitaryanscn  filename is ok.
Source: https://dictionary.cambridge.org/dictionary/english/filename